.vib/kubeapps/goss/goss.yaml

```diff
@@ -0,0 +1,30 @@
+http:
+  http://kubeapps-internal-kubeappsapis:{{ .Vars.kubeappsapis.service.ports.http }}/core/plugins/v1alpha1/configured-plugins:
+    status: 200
```

Comment:
I'm missing a test for the dashboard service, could it be added here?

Reply:
added